Temperature_API/models.py

```diff
@@ -0,0 +1,15 @@
+from sqlalchemy import Column, Integer, String, DateTime, ForeignKey
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class Temperature(Base):
+    __tablename__ = "temperature"
+
+    id = Column(Integer, primary_key=True, index=True)
+    date_time = Column(DateTime)
```

Comment:
Consider setting `nullable=False` for the `date_time` column if it is a required field. This will ensure that every temperature record has a timestamp.